sage-ios/SAGE/SAGE/LoginController.swift

```diff
@@ -35,35 +37,67 @@ class LoginController: UIViewController, UITextFieldDelegate {
         })
     }
     
-    func textFieldShouldReturn(textField: UITextField) -> Bool {
+    //
+    // MARK: - Methods to handle navigation
+    //
+    
+    func pushRootTabBarController() {
+        LoginHelper.setUserSingleton()
+        let rootTabBarController = RootTabBarController()
+        self.presentViewController(rootTabBarController, animated: false, completion: nil)
+    }
+    
+    func showError(message: String) {
```

Comment:
might wanna make a catgory to UIViewController and make this a method instead so it doesnt need to be reimplemented every time